### PR TITLE
Feature/ssd erase 2

### PR DESCRIPTION
**Pull request by hh1012**
ssd read test 강화
- 조금더 edge case에 가깝도록 값을 변경하였습니다.

ssd erase 개발을 위한 UT 추가

write.h 추가

### CODE REVIEW

File: SSD_Excutor/erase_test.cpp [hunk 1]
```diff
@@ -1,9 +1,49 @@
...
+    EXPECT_EQ(readValue, expectErasedValue);
+    
+    readCommand.execute(addressTwo);
+    readValue = std::stoul(ssdHelper.getReadResultFromFile(), nullptr, 16);
+    EXPECT_EQ(readValue, expectErasedValue);
+    
+    readCommand.execute(addressThree);
+    readValue = std::stoul(ssdHelper.getReadResultFromFile(), nullptr, 16);
+    EXPECT_EQ(readValue, expectErasedValue);
+}
```
Comment (uiyangco) [inline SSD_Excutor/erase_test.cpp:49]: UT 강화하신 거 좋네요!

**Review by uiyangco — APPROVED** [review]
